Mathlib/Topology/UniformSpace/Basic.lean

```diff
@@ -1120,6 +1120,15 @@ nonrec theorem UniformContinuous.comp [UniformSpace β] [UniformSpace γ] {g : 
   hg.comp hf
 #align uniform_continuous.comp UniformContinuous.comp
 
+/--If a function `T` is uniformly continuous in a uniform space `β`,
+then its `n`-th iterate `T^[n]` is also uniformly continuous.-/
+theorem uniformContinuous_ite [UniformSpace β] (T : β → β) (n : ℕ) (h : UniformContinuous T) :
```

Comment:
The name is bad, because `ite` means `if then else`. What about `UniformContinuous.iterate`?

Reply:
Ok.